conversation_reconstruction_dataflow/conversation_reconstruction/setup.py

```diff
@@ -102,20 +98,35 @@ def run(self):
 # so this dependency will not trigger anything to be installed unless a version
 # restriction is specified.
 REQUIRED_PACKAGES = [
-    'google-cloud >= 0.27.0',
-    'google-cloud-storage >= 1.3.2',
+    'google-cloud == 0.28.0',
+    'google-cloud-bigquery== 0.28.0'
+#    'google-cloud-storage == 1.3.2',
+#    'google-apitools == 0.5.10' 
     ]
 
 setuptools.setup(
     name='construct_utils',
     version='0.0.1',
-    description='A package to ingest Wikipedia comments.',
+    description='A package to reconstruct Wikipedia conversations.',
     install_requires=REQUIRED_PACKAGES,
-    packages=setuptools.find_packages(),
-    cmdclass={
+    packages=setuptools.find_packages()#,
+#    cmdclass={
+        # Command class instantiated and run during pip install scenarios.
+#        'build': build,
+#        'CustomCommands': CUSTOM_COMMANDS,
+#        }
+    )
+
+setuptools.setup(
+    name='mwparserfromhell',
+    version='0.0.1',
+    description='A package to parse Wikipedia talk pages.',
+    install_requires=[],
+    packages=setuptools.find_packages()#,
+#    cmdclass={
         # Command class instantiated and run during pip install scenarios.
-        'build': build,
-        'CustomCommands': CustomCommands,
-        }
+#        'build': build,
+#        'CustomCommands': [['python', 'setup.py', 'install']],
```

Comment:
lots of random looking commented out stuff here.

Reply:
cleaned.